pkg/controllers/provisioning/v1alpha1/reallocation/suite_test.go

```diff
@@ -72,7 +71,6 @@ var _ = Describe("Reallocation", func() {
 			},
 			Spec: v1alpha1.ProvisionerSpec{
 				Cluster:    &v1alpha1.ClusterSpec{Name: "test-cluster", Endpoint: "http://test-cluster", CABundle: "dGVzdC1jbHVzdGVyCg=="},
-				TTLSeconds: ptr.Int32(300),
```

Comment:
Does your test rely on the TTL behavior at all? Will it change tests to have nodes instantly deleted after underutilized?

Reply:
At the end of the first test in this suite, it checks to see if the TTL annotation is part of the node. Therefore, if our default logic fails here, the test will fail.